mlflow/server/js/src/experiment-tracking/components/ExperimentListView.tsx

```diff
@@ -392,6 +391,11 @@ const classNames = {
     justifySelf: 'end',
     marginRight: '10px',
   },
+  icon: (theme: Theme) => ({
+    color: theme.colors.actionDefaultTextDefault,
+    fontSize: 24,
```

Comment:
(should evaluate to `24` as well)
```suggestion
    fontSize: theme.general.iconSize,
```